### ISSUE TITLE
app crasht

### ISSUE THREAD

**Issue by paulus644** [RWensveen/com.broadlink#7]
first off all    great app 

i have the rm2 broadlink
but when i  start a flow the app crashes..............flow   .......when i say someting ......verstuur commando ....cmd1
what i am doing wrong

**Comment by RWensveen** [comment]
Hello,
Thanks for liking the app :-)

About the crash: there was an error in the flow control (it could not distinguish between multiple devices of the same type).
If you paired the device with the previous version of the app, an now use the new (corrected) one, this might happen.

Have you tried to remove the RM2 device and its flows, and then adding it again? Not something you would normally do when
an app is updated (at least, I do not), but in this case it might solve the problem.

The tests I did with a RM3 Mini worked ok. but I removed the device and flows first, so I may not have seen the issue.
